Sources/FaceLiveness/Views/Liveness/FaceLivenessDetectionViewModel+FaceDetectionResultHandler.swift

```diff
@@ -84,9 +84,12 @@ extension FaceLivenessDetectionViewModel: FaceDetectionResultHandler {
     }
 
     func handleNoMatch(instruction: Instructor.Instruction, percentage: Double) {
+        let noMatchTimeoutInterval: TimeInterval = 7
         self.livenessState.awaitingFaceMatch(with: instruction, nearnessPercentage: percentage)
-        noMatchCount += 1
-        if noMatchCount >= 210 {
+        if noMatchStartTime == nil {
+            noMatchStartTime = Date()
+        }
+        if let elapsedTime = noMatchStartTime?.timeIntervalSinceNow, abs(elapsedTime) >= noMatchTimeoutInterval {
```

Comment:
do we have an accompanying test for the criteria of no match?

Reply:
added unit test to check for no match timeout handling.